CakePHP/Sniffs/NamingConventions/ValidVariableNameSniff.php

```diff
@@ -14,16 +14,16 @@
  * @since         CakePHP CodeSniffer 0.1.1
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
+namespace CakePHP\Sniffs\NamingConventions;
 
-if (class_exists('PHP_CodeSniffer_Standards_AbstractVariableSniff', true) === false) {
-    throw new PHP_CodeSniffer_Exception('Class PHP_CodeSniffer_Standards_AbstractVariableSniff not found');
-}
+use PHP_CodeSniffer\Sniffs\AbstractVariableSniff;
+use PHP_CodeSniffer\Files\File;
 
 /**
  * Checks the naming of variables and member variables.
  *
  */
-class CakePHP_Sniffs_NamingConventions_ValidVariableNameSniff extends PHP_CodeSniffer_Standards_AbstractVariableSniff
+class ValidVariableNameSniff extends AbstractVariableSniff
```

Comment:
This sniffs seems to be based off `Zend.NamingConventions.ValidVariableName`. So instead of maintaining our own class we can directly include that rule in our `ruleset.xml` and just exclude the couple of checks it has regarding leading underscores.